sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -255,3 +287,43 @@ case class AlterViewAsCommand(
     session.sessionState.catalog.alterTable(updatedViewMeta)
   }
 }
+
+/**
+ * Drops a view which matches the given name.
+ *
+ * If the view name contains database prefix, this command will drop a permanent view matching the
+ * given name, or throw an exception if view not exist. Else, this command will try to drop a local
+ * temporary view first, if view not exist, try global temporary view next, if still not exist, try
```

Comment:
Yes. Right. Up to now, all happens in the same session. So, temporary view is created after permanent view always. It works like STACK.

However, this is a global view made by other session. So, it does not work like STACK. Even in the same session, if we execute 'create temporary view' and 'create global view', we must delete the local temporary view first. So, it seems more serious to me.